src/ripple/shamap/impl/SHAMapSync.cpp

```diff
@@ -419,7 +419,7 @@ SHAMapAddNode SHAMap::addRootNode (Blob const& rootNode,
 
     assert (seq_ >= 1);
     auto node = SHAMapAbstractNode::make(
-        rootNode, 0, format, uZero, false, f_.journal ());
+        rootNode, 0, format, SHAMapHash{uZero}, false, f_.journal ());
```

Comment:
I missed that you were talking about a file-scope static.  I thought you meant function scope.  I'll look into the code generation...